ci/patch_src_in_place

```diff
@@ -17,9 +23,14 @@ if [ -x $CP ]
 then
     for FILE in $(git ls-tree --full-tree --name-only HEAD -r src | grep -v -e src/control/vendor -e .go\$ -e pb-c -e debug_setup.h)
     do
-        $CP --quiet --no-tree -file --show-types --no-summary --ignore "$def_ignore" --strict "$FILE" --fix-inplace
+	ignore="$def_ignore"
+	grep -lq CRT_RPC_DECLARE "$FILE"
+	if [ $? -eq 0 ]
+	then
```

Comment:
I'm surprised shellcheck didn't complain about this.  Maybe that only happens in newer versions of it.  We really should be linting on Fedora where much newer tools are.  In any case, the _idiomatic_ way to write this is:
```suggestion
	if grep -lq CRT_RPC_DECLARE "$FILE"; then
```